### ISSUE TITLE
Replace checkbox for locking a layer by an icon besides the layer image

### ISSUE THREAD

**Issue by SebastianZ** [firebug/pixel-perfect#9]
Instead of having a _Lock_ checkbox within the layer options there should be a lock symbol besides the image in the layer list below the option for toggling the visibility.

A bigger version of the [lock symbol used within Firebug's _DOM_ panel](https://github.com/firebug/firebug/blob/master/extension/skin/classic/readOnlyProp.svg) would fit well here: ![Lock symbol used within Firebug's *DOM* panel](https://cloud.githubusercontent.com/assets/958943/6428401/290cb55a-bfa1-11e4-9864-2ebeb04b7f08.png) (Good, that I created it as SVG! :-) )

Sebastian

**Comment by mbuckley** [comment]
![1 home](https://cloud.githubusercontent.com/assets/788414/6547093/0cd7afbc-c58b-11e4-9861-cd0bd8fefc32.png)

**Comment by aliuk2012** [comment]
Possible a fourth one for cloning  issue #36 

I like the directional arrows but would be a hassle if you what to move it to a specific x/y location

**Comment by mbuckley** [comment]
Good idea and agreed that the clone would belong somewhere over with the overlay. Either as a primary button or as a hover action similar to how delete works.

@aliuk2012 I guess I was looking at it from a dragging first, then adjusting point of view. If going to a specific x/y would be a common use-case we could still keep the x/y as inputs for large movements.

x: <input>, y: <input> and place that just above the directional arrows.

**Comment by SebastianZ** [comment]
While I agree with the points mentioned here, they are unrelated to this issue. This one really just covers replacing the locking checkbox by an icon besides the layer image.

Replacing the visibility checkbox is covered in #8, cloning is covered in #36, the positioning arrows are unrelated (and should be discussed in their own issue). And if an icon is really wished for the inverting the image, this should be done in a new issue.

Sebastian

**Comment by janodvarko** [comment]
> they are unrelated to this issue. This one really just covers replacing the locking checkbox
> by an icon besides the layer image.

I don't think it's necessary to block the discussion and force spreading across the issue list.

Honza
